bloom-housing/bloom
# fix: enable utilities and accessibility features

This PR addresses #4440 #4439 


## Description

Listing utilities and accessibility features functionality already exist. However it was not working correctly. This PR fixes all of the bugs as well as connecting it to the new feature flag functionality

NOTE: the jurisdiction table fields have also been removed since we can now use the feature flag table

## How Can This Be Tested/Reviewed?

Both feature flags are added to the staging seeded data for "Bloomington" jurisdiction so the following can be tested with a new reseed

- Create a new listing in partners for the "Bloomington" jurisdiction.
  - Verify that the Utilities checkbox section and the accessibility features checkbox section appear
  - Verify that you can select specific utilities and accessibility features. On save make sure they are saved correctly
- Edit the listing and change the utilities/accessibility features
  - Verify that the fields are correctly updated on save
- Create a new listing not for "Bloomington" jurisdiction and verify that the two sections don't appear
- Go to the public site and verify that the first listing created shows the correct utilities and accessibility features on the listing details page

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
